Templates/TimerTrigger-Powershell/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "defaultFunctionName": "TimerTriggerPowershell",
+    "description": "A Powershell function that will be run on a specified schedule",
+    "name": "TimerTrigger - Powershell",
+    "language": "Powershell",
+    "category": [
+        "Experimental",
+        "Data Processing"
```

Comment:
There is was not a very string argument against putting time trigger templates is data processing. The closes probable use of time triggers would be to fetch or process data periodically. In time we would have a schedule category in templates. All C# and Node templates are there in core unless the binding itself needs work.